### DISCUSSION TITLE
[IntelliSense] add Intellisense in angular routerLinkActive attribute

### DISCUSSION THREAD

**Discussion by DimitriTsikaridze** [tailwindlabs/tailwindcss#12719] [Ideas]
currently routerLinkActive="text-red-500" won't give you any intellisense even tho routerLinkActive directive expects css class name.

![image](https://github.com/tailwindlabs/tailwindcss/assets/56136563/e9cf9c28-74b5-49ad-b3c8-4857ba3697e5)

**Comment by wongjn** [comment]
You could try configuring the [`tailwindCSS.experimental.classRegex` setting](https://github.com/tailwindlabs/tailwindcss/issues/7553#issuecomment-735915659).

**Comment by ezhupa99** [comment]
Here is the solution for VSCode

append `Tailwind CSS: Class Atributes` by adding `routerLinkActive' in the list

![image](https://github.com/user-attachments/assets/999b12ad-dd8a-4c45-9cf2-9c7228030e04)

**Reply by DimitriTsikaridze** [reply]
this works, I should've looked into tailwind config options further thanks.
